docs/02-app/02-api-reference/08-next-cli.mdx

```diff
@@ -184,16 +328,93 @@ You can also run `next info --verbose` which will print additional information a
 provides a guided setup to install any required dependencies if ESLint is not already configured in
 your application.
 
-If you have other directories that you would like to lint, you can specify them using the `--dir`
-flag:
+To get a list of the available options with `next lint`, run the following command inside your project directory:
+
+```bash filename="Terminal"
+npx next lint -h
+```
+
+The output should look like this:
+
+```bash filename="Terminal"
+Usage: next lint [directory] [options]
+
+Runs ESLint for all files in the `/src`, `/app`, `/pages`, `/components`, and `/lib`
+directories. It also provides a guided setup to install any required dependencies if ESLint
+is not already configured in your application.
+
+Arguments:
+  [directory]                            A base directory on which to lint the application.
+                                         If no directory is provided, the current
+                                         directory will be used.
+
+Options:
+  -d, --dir, <dirs...>                   Include directory, or directories, to run ESLint.
+  --file, <files...>                     Include file, or files, to run ESLint.
+  --ext, [exts...]                       Specify JavaScript file extensions. (default:
+                                         [".js", ".mjs", ".cjs", ".jsx", ".ts", ".mts", ".cts", ".ts
+x"])
+  -c, --config, <config>                 Uses this configuration file, overriding all other
+                                         configuration options.
+  --resolve-plugins-relative-to, <rprt>  Specify a directory where plugins should be
+                                         resolved from.
+  --strict                               Creates a `.eslintrc.json` file using the Next.js
+                                         strict configuration.
+  --rulesdir, <rulesdir...>              Uses additional rules from this directory(s).
+  --fix                                  Automatically fix linting issues.
+  --fix-type <fixType>                   Specify the types of fixes to apply (e.g., problem,
```

Comment:
```suggestion
  --fix-type <fixType>                   Specify the types of fixes to apply (e.g. problem,
```

Reply:
Grammarly taught me to use a comma after a `e.g.` <picture data-single-emoji=":lol:" title=":lol:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/lol/5986e156eca2154f.png" alt=":lol:" align="absmiddle"></picture>  → https://www.bachelorprint.com/commas/comma-before-or-after-e-g/#:~:text=The%20abbreviation%20%E2%80%9Ce.g.%E2%80%9D%20stands%20for,It%20is%20parenthetic.